WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationTracker.kt

```diff
@@ -4,9 +4,114 @@ import org.wordpress.android.analytics.AnalyticsTracker
 import javax.inject.Inject
 import javax.inject.Singleton
 
+const val ORIGIN_VERTICALS_FULLSCREEN_ERROR = "Verticals - fullscreen"
+const val ORIGIN_VERTICALS_LIST_ITEM_ERROR = "Verticals - list item"
+const val ORIGIN_SEGMENTS_ERROR = "Segments"
+const val ORIGIN_DOMAINS_ERROR = "Domains"
+const val ORIGIN_PREVIEW_ERROR = "Preview"
+
 @Singleton
 class NewSiteCreationTracker @Inject constructor() {
-    fun trackSiteCreated() {
-        AnalyticsTracker.track(AnalyticsTracker.Stat.CREATED_SITE)
+    fun trackSiteCreationAccessed() {
+        AnalyticsTracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_ACCESSED)
```

Comment:
Would there be any value in injecting the `AnalyticsTrackerWrapper` and use that instead of directly using `AnalyticsTracker`? Since you have already wrapped everything in one injected class, the testing its usage is a non-issue and there is not really anything to test in this class, but I wanted to bring it up in case I am missing anything and you weren't aware of `AnalyticsTrackerWrapper`. (I wasn't)

Reply:
👍  makes sense. 

Fixed in [#2b246f2](https://github.com/wordpress-mobile/WordPress-Android/pull/8998/commits/2b246f2f464dee6b6266e8df69b031a0148cbf52)